ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -471,7 +473,7 @@ private int executeIncrementalLoad(DriverContext driverContext) {
         if (work.hasBootstrapLoadTasks()) {
           LOG.debug("Current incremental dump have tables to be bootstrapped. Switching to bootstrap "
                   + "mode after applying all events.");
-          return executeBootStrapLoad(driverContext);
+          return executeBootStrapLoad(driverContext, true);
```

Comment:
Can we set isBootstrapDuringIncLoad in ReplLoadWork object constructor itself? so that need not pass it as extra argument and need not set it after each iteration.